packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -244,8 +244,10 @@ class RenderStack extends RenderBox
   /// top left corners.
   RenderStack({
     List<RenderBox> children,
-    FractionalOffset alignment: FractionalOffset.topLeft
-  }) : _alignment = alignment {
+    FractionalOffset alignment: FractionalOffset.topLeft,
+    bool clipOverflowingChildren: true
```

Comment:
I'd make this an enum called `Overflow` with values `clip` and `visible` (borrowing terminology from https://developer.mozilla.org/en-US/docs/Web/CSS/overflow)